be/src/agent/pusher.cpp

```diff
@@ -73,7 +72,7 @@ AgentStatus Pusher::init() {
     // Set download param
     string tmp_file_dir;
     status = _get_tmp_file_dir(olap_table->storage_root_path_name(), &tmp_file_dir);
-    if (status != DORIS_SUCCESS) {
+    if (status != Status::OK()) {
         LOG(WARNING) << "get local path failed. tmp file dir: " << tmp_file_dir;
```

Comment:
print `st.get_error_msg()`

Reply:
I have already removed pusher.cpp file in next pr.